### PR TITLE
Add dedicated php child process endpoint

### PR DESCRIPTION
**Pull request by gwleuverink**
As discussed this PR accompanies https://github.com/NativePHP/electron/pull/129

I've updated all related tests, but one concerning the Window component fails still. Did not touch it since it was unrelated to this PR. Will do a sweep for other tests when time allows it.

Keeping this in draft since I want to do some more testing on the electron side

### CODE REVIEW

**Comment by gwleuverink** [comment]
I've also fixed the window test. See comments for details

File: tests/Windows/WindowTest.php [hunk 1]
```diff
@@ -43,7 +46,7 @@
     expect($windowArray['maximizable'])->toBeTrue();
     expect($windowArray['closable'])->toBeTrue();
     expect($windowArray['fullscreen'])->toBeTrue();
-    expect($windowArray['kiosk'])->toBeFalse();
+    expect($windowArray['kiosk'])->toBeTrue();
```
Comment (gwleuverink) [inline tests/Windows/WindowTest.php:49]: The default value for `->kiosk()` is `true` in `Window.php`

File: tests/Windows/WindowTest.php [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 use Native\Laravel\Facades\Window;
 use Native\Laravel\Windows\Window as WindowClass;
 
 it('test window', function () {
+    Http::fake();
     Window::shouldReceive('open')
         ->andReturn(new WindowClass('main'));
 
     $window = Window::open()
+        ->setClient(new Client)
```
Comment (gwleuverink) [inline tests/Windows/WindowTest.php:13]: the `title()` method in `Window.php` calls the Client internally, which isn't set at that point

Comment (simonhamp) [reply tests/Windows/WindowTest.php:13]: This is fine for now. We can work out the dependencies for better testing later

**Review by simonhamp — APPROVED** [review]
Love it. It's great to see tests passing again!

**Pull request merged**